### PR TITLE
add readme for salary service

### PR DESCRIPTION
**Pull request by MalarCB**
[Jira Card](https://careerbuilder.atlassian.net/browse/RDP-357)

### CODE REVIEW

File: SalaryService.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+Requests consist of:
+
+| Parameter | Require/Optional | Type | Description |
+|:----------|:-------|:-------|:-------|
+|`carotene_id`| Yes | string | A unique id|
+|`cbsa_code` | Optional | string |  Unique Identifiers for US Metropolitan Statistical areas. |
+|`postal_code` | Optional | string |  Optional. The post code, postal code, or ZIP Code of an address.|
+|`salary_period` | Yes | string | Possible values are "Year", "Hour". | 
+|`country` | Yes | string |A country name or two-letter ISO-3166 country code. Supports only "US". |
```
Comment (glpayson) [inline SalaryService.md]: I don't think we have any intention of ever supporting full country names (at least not currently). We should probably just say 'An ISO-3166 country code.'

File: SalaryService.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The Salary service is available at `/core/salary`.
+
+Request Structure
+-----------
+
+Requests consist of:
+
+| Parameter | Require/Optional | Type | Description |
+|:----------|:-------|:-------|:-------|
+|`carotene_id`| Yes | string | A unique id|
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
|`carotene_id`| Yes | string | A unique id representing a [carotene job title classification|https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/JobTitle.md#taxonomies]|
```

File: SalaryService.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+|`cbsa_code` | Optional | string |  Unique Identifiers for US Metropolitan Statistical areas. |
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
|`cbsa_code` | Optional | string |  Unique Identifiers for US metropolitan statistical areas. |
```

File: SalaryService.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+|`postal_code` | Optional | string |  Optional. The post code, postal code, or ZIP Code of an address.|
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
|`postal_code` | Optional | string ||
```

Comment (glpayson) [reply SalaryService.md]: I'd just omit the description here. If people don't know what postal codes are then they probably have much larger problems to deal with than figuring out how to use some API.

File: SalaryService.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+_____________
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+Summary
+-----------
+The Salary service returns salary info for a carotene id and, optionally, a postal or a cbsa code in either hourly or yearly amounts.
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
The Salary service returns salary info for a carotene id and, optionally, a postal or a CBSA code. Salary info can be requested in either hourly or yearly amounts.
```

Comment (glpayson) [reply SalaryService.md]: It is maybe a bit ambiguous if your last prepositional phrase is modifying  'salary info', 'cbsa coda', 'postal code', etc.

File: SalaryService.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  https://api.careerbuilder.com/core/salary \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "43.1",
+	"salary_period": "year",
+	"postal_code": "",
+	"cbsa_code": "",
+	"country": "US"
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
	"carotene_id": "43.1",
	"salary_period": "year",
	"country": "US"
```

File: SalaryService.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+  https://api.careerbuilder.com/core/salary \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "43.1",
+	"postal_code": "77494",
+	"cbsa_code": "",
+	"salary_period": "year",
+	"country": "US"
```
Comment (glpayson) [inline SalaryService.md]:
```suggestion
	"carotene_id": "43.1",
	"postal_code": "77494",
	"salary_period": "year",
	"country": "US"
```

File: SalaryService.md (hunk 6 above)
Comment (glpayson) [reply SalaryService.md]: Oops I accidentally committed this suggestion...

File: SalaryService.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+which consists of `currency`, `percentile_10`, `percentile_90`, `period`.
+```json
+{
+  "data": {
+    "currency": "USD",
+    "percentile_10": 21870.0,
+    "percentile_90": 59100.0,
+    "period": "YEAR"
+  }
+}
```
Comment (glpayson) [inline SalaryService.md:112]: I think we should mention that the `currency` field is ISO 4217 and the possible `period` values are `HOUR` and `YEAR` depending on the `salary_period` on the request.

Comment (glpayson) [reply SalaryService.md:112]: ... and I just noticed that we were inconsistent in naming those two fields. Is it too late to fix that?

File: SalaryService.md (hunk 4 above)
Comment (MalarCB) [reply SalaryService.md]: Good point but leaving empty felt odd to me so added "a postal code"

File: SalaryService.md (hunk 8 above)
Comment (MalarCB) [reply SalaryService.md:112]: Not too late as Dimitry has not implemented yet.

**Review by glpayson — APPROVED** [review]

**Review by DavidGWu — APPROVED** [review]

**Pull request merged**